AonWeb.FluentHttp/Handlers/Caching/CacheConfigurationHandlerCore.cs

```diff
@@ -123,13 +124,40 @@ protected ICacheContext GetContext(IHandlerContext handlerContext)
             context.Items["CacheHandlerCachedItem"] = result;
         }
 
-        protected async Task TryGetRevalidatedResult(IHandlerContextWithResult handlerContext, HttpRequestMessage request, HttpResponseMessage response)
+        protected async Task TryGetNotModifiedResult(IHandlerContextWithResult handlerContext,HttpRequestMessage request, HttpResponseMessage response)
         {
             if (!Enabled)
                 return;
 
             var context = GetContext(handlerContext);
 
+            if (response.StatusCode == HttpStatusCode.NotModified)
+            {
+                var cachedResult = await _cacheManager.Get(context);
```

Comment:
You should be able to access the item gotten from cache via context.Items["CacheHandlerCachedItem"] from line 124